elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -277,6 +280,22 @@ export abstract class BaseTransaction {
 			);
 		}
 
+		// Skip for type 4 due to dynamic nature of fee for multi-signature. Validation happens on multisignature class.
```

Comment:
Even though the validation happens inside the `MultisignatureTransaction` class, I think we should have `public static FEE = MULTISIGNATURE_FEE.toString();` declared inside the class as well.

Reply:
Wouldn't it be misleading as it will actually not be used at all?

Reply:
I would replace `MULTISIGNATURE_FEE` by `public static FEE` inside `4_multisignature_transaction.ts` to keep consistency with the other transaction types.

Reply:
Changed to use `MultisignatureTransaction.FEE` instead of constant.